ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2162,6 +2162,13 @@ router_ospf:
               - < IPv4 subnet / netmask >
               - < IPv4 subnet / netmask >
             prefix_list: < prefix list name >
+        < area >:
+          type: < normal | stub | nssa | default -> normal >
```

Comment:
`normal` is not an option

Reply:
this should be just `type: < stub | nssa >

Reply:
Using non-existing type default is not breaking. On the other hand it's good for readability to highlight that area type is not necessarily stub or nssa. I'd suggest to keep as is.